examples/darwin-framework-tool/templates/tests/ciTests.json

```diff
@@ -63,6 +63,10 @@
         "Test_TC_LWM_3_1",
         "Test_TC_LWM_3_2",
         "Test_TC_LWM_3_3",
+        "Test_TC_OTCCM_2_1",
```

Comment:
This test does not use EqualityCommands?  Why is it disabled, and why is the disabling claiming it's because of lack of support for EqualityCommands?